projects/Mallard/src/screens/settings-screen.tsx

```diff
@@ -140,12 +147,27 @@ const SettingsScreen = ({ navigation }: NavigationInjectedProps) => {
             signedIn: data => [
                 {
                     key: `Sign out`,
-                    title: `Sign out of ${data.userDetails.publicFields.displayName}`,
+                    title: data.userDetails.publicFields.displayName,
                     data: {
                         onPress: async () => {
                             await signOut()
                         },
                     },
+                    proxy: (
+                        <Text
+                            style={{ color: '#41A9E0', ...getFont('sans', 1) }}
```

Comment:
can we map this color to the palette and move this inline style to stylesheet?

Reply:
I have no idea what colour this is in the palette - can you provide?

Reply:
Will move into Stylesheet.

Reply:
think this is ben's new blue. can you drop it into a variable in `colors.ts`? color.ocean or `color.supportBlue` or anything really. It's used all over the onboardings so good to scope it

Reply:
Done both.